modules/tinymce/src/core/main/ts/api/html/Node.ts

```diff
@@ -452,6 +452,23 @@ class AstNode {
     return collection;
   }
 
+  /**
+   * Get all children of the current node.
```

Comment:
```suggestion
   * Get all children of this node.
```
We changed current to this above, so I assume we should do the same here?